gulpfile.js

```diff
@@ -197,3 +198,21 @@ gulp.task("copy-licenses-octicons", () =>
 gulp.task("copy-licenses-ofl", () =>
     gulp.src(config.copy.licenses_ofl.sources)
         .pipe(copy(config.copy.licenses_ofl.dest, {prefix: 1})));
+
+// Validate contents
+
+gulp.task("validate", () => {
+    const paths = [
+        config.less.dest,
+        config.copy.fonts.dest,
+        config.copy.octicons.dest,
+    ];
+
+    for (const path of paths) {
+        fs.stat(path, (err) => {
```

Comment:
We might want to use statSync() here instead? Otherwise I don't think gulp will notice the callback throwing, even if it happens before gulp finishes.